src/Components/header/header.js

```diff
@@ -26,17 +36,21 @@ export const IconContainer = styled.div`
   align-items: center;
   justify-content: center;
   font-size: 1.2rem;
-  color: white;
+  color: ${(props) => props.theme.text};
 `;
 
 export const Header = () => {
+  const { theme, toggleTheme } = useTheme();
+
   return (
     <HeaderContainer>
-      <Text size="1.5rem">devfinder</Text>
-      <RightContainer>
-        <Text>Light</Text>
+      <Text color={theme.text} size="1.5rem">
```

Comment:
The sizes could be moved into a constants file similar to previous projects if they're being repeated in multiple components